src/can_common.h

```diff
@@ -1,11 +1,13 @@
 #ifndef _CAN_COMMON_H_
 #define _CAN_COMMON_H_
 
-#define CAN_THROTTLE 45
-#define CAN_MOTOR_ENABLE 20
+#define CAN_THROTTLE      45
+#define CAN_MOTOR_ENABLE  20
 #define CAN_ACCESSORIES
-#define CAN_TELEMETRY
+#define CAN_TELEMETRY     0xA0
 
-#define CAN_BMS // TODO we have to split these out
+// Assuming default BMS node ID of 0x01
```

Comment:
You could make this a macro that takes the node ID as an argument so that we're not hardcoding the ID. We can then add a macro for `CAN_BMS_DEFAULT_ID` so we have a known value to work off of.

Reply:
Done!

Reply:
That works, though I was thinking of using a [macro argument](https://www.cs.yale.edu/homes/aspnes/pinewiki/C(2f)Macros.html) instead.

Reply:
Oh that makes more sense. I've fixed it